ReactWindows/ReactNative.Shared/Common/LifecycleState.cs

```diff
@@ -17,9 +17,9 @@ public enum LifecycleState
         BeforeResume,
```

Comment:
>BeforeResume [](start = 8, length = 12)

If we're making breaking changes to this public API anyway (I'm comfortable with this), maybe we should just call this "Suspended" to better align with the UWP terminology.